src/context/chartLayoutContext.tsx

```diff
@@ -14,6 +14,7 @@ import { TooltipContextProvider, TooltipContextValue } from './tooltipContext';
 export const XAxisContext = createContext<XAxisMap | undefined>(undefined);
 export const YAxisContext = createContext<YAxisMap | undefined>(undefined);
 export const PolarAngleAxisContext = createContext<BaseAxisMap | undefined>(undefined);
+export const PolarRadiusAxisContext = createContext<BaseAxisMap | undefined>(undefined);
```

Comment:
I wonder how much overlap is there between the PolarAngleAxis and PolarRadiusAxis?

Reply:
they're basically the same in terms of the props they get back. What I'm not sure about is if the axis map comes back differently from the generator function or not. 

but I didn't want to mess with combining them just yet. Can do that later if we need